src/scriptwriter/scenarios/ScenariosHandler.js

```diff
@@ -110,4 +114,21 @@ export default class ScenariosHandler {
 
         return scenario;
     }
+
+    /**
+     * Loads a start url for scenario from specified url paths
+     * @returns {string} start url
+     */
+    _getRandomScenarioStartUrl() {
+        let scenarios = this._config.urlPaths;
```

Comment:
This is quite confusing, since you are getting variable `urlPaths` and suddenly start calling it `scenarios`. You can even use object destructuring if you use the original name.

```javascript
const { urlPaths } = this._config;
```